packages/eslint-config-next/index.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * This shareable configuration uses @rushstack/eslint-patch to include its plugins
+ * within as dev dependencies instead of imposing them as peer dependencies
+ *
+ * https://www.npmjs.com/package/@rushstack/eslint-patch
+ */
+
+require('@rushstack/eslint-patch/modern-module-resolution')
+
+module.exports = {
+  extends: [
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+    'plugin:@next/next/recommended',
+  ],
+  plugins: ['import', 'react'],
+  rules: {
+    'import/no-anonymous-default-export': 'warn',
+    'react/react-in-jsx-scope': 'off',
+  },
+  parser: '@babel/eslint-parser',
+  parserOptions: {
+    requireConfigFile: false,
+    sourceType: 'module',
+    allowImportExportEverywhere: true,
+    babelOptions: {
+      presets: ['next/babel'],
+    },
+  },
+  overrides: [
+    {
+      files: ['**/*.ts?(x)'],
+      parser: '@typescript-eslint/parser',
+      parserOptions: {
+        sourceType: 'module',
+        ecmaFeatures: {
+          jsx: true,
+        },
+        warnOnUnsupportedTypeScriptVersion: true,
+      },
+    },
+  ],
+  settings: {
+    react: {
+      version: 'detect',
+    },
```

Comment:
```suggestion
    },
    'import/parsers': {
      [require.resolve('@typescript-eslint/parser')]: ['.ts', '.tsx', '.d.ts'],
    },
    'import/resolver': {
      [require.resolve('eslint-import-resolver-node')]: {
        extensions: ["<extensions here>"],
      },
    },
```

This is needed to ensure the import plugin is able to find the resolvers, otherwise it will resolve it relative to the users files which is not guaranteed to work

Reply:
I assume you forgot to include the actual file extensions?

I updated it to `extensions: ['.js', '.jsx', '.ts', '.tsx']` 👍

Reply:
I was unsure which extensions you actually wanted to apply it to (css, less, sass etc) so I left it "blank" 👍